crates/sui/src/sui_commands.rs

```diff
@@ -764,3 +1054,55 @@ fn read_line() -> Result<String, anyhow::Error> {
     io::stdin().read_line(&mut s)?;
     Ok(s.trim_end().to_string())
 }
+
+#[allow(unused)]
+async fn start_simple(config: Option<PathBuf>, no_full_node: bool) -> Result<(), anyhow::Error> {
```

Comment:
What's the plan for this function? If we intend to leave it some comments should be added to explain

Reply:
Right, I plan to remove this before landing, this is just how `sui start` worked before in the orig code (I cross check it to make sure I don't break things, so I kept it in the same file for making it easier to look at).

Reply:
Removed.